nfd.package.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: REPLACE_NAMESPACE
+
```

Comment:
Hmm, what is this file for?

Reply:
is part of the OLM,that I am going to move to the operator frame work

Reply:
Seems, that it was dropped, 👍